drydock/values.yaml

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
@sh8121att This is where we need to understand and define any input drydock needs (at the application level).

Today, my understanding is it needs MaaS connectivity information and API keys.  If we had a sample of that, we could provide the interface here. 

These should align with likely the same inputs @larryrensing maas chart has if these can be set there, in terms of "defaults"

We may also want to add dependencies here - what does drydock depend on? MaaS? Postgre? At some future point, the datamanager service (does not exist yet).